EmbeddingOnlyScearioBase.cs

```diff
@@ -28,21 +28,21 @@ public WikiCohereEmbeddingOnlyDocument(string id, float[] embedding)
         }
     }
 
-    abstract class WikiCohereEmbeddingOnlyBaseSceario : Scenario
+    abstract class EmbeddingOnlyScearioBase : Scenario

```

Comment:
This base class is for non-streaming scenario only but current name does not reflect that. Maybe rename or add a comment at least?

Reply:
Also since this file has reference to BigANNBinaryFormat, we should probably reflect that this is for mebedding only for binary format.

Reply:
not sure what should be a better name ? BigANNBinaryEmbeddingOnlyScearioBase